zeebe/qa/integration-tests/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>io.camunda</groupId>
     <artifactId>zeebe-qa</artifactId>
-    <version>8.5.16</version>
+    <version>8.5.16-SNAPSHOT</version>
```

Comment:
❓ I assume you've checked but just to be sure: is this the only place in the repo on this branch that suffers from this? That's super weird then.

Reply:
It's basically the qa module, right?